game/src/test/scala/io/aigar/game/FoodSpec.scala

```diff
@@ -0,0 +1,17 @@
+package io.aigar.game
+
+import io.aigar.game.serializable.Position
+import org.scalatest._
+
+
+class FoodSpec extends FlatSpec with Matchers{
+  //Very dummy test right now. Will update it after I do the Resources spawn / respawn
+  "Food" should "generate a state with the predefined regular, silver and gold resources in it" in {
```

Comment:
I don't know, since it will be updated once they'll be a spawning function? I can remove it.